packages/app/src/cli/services/app/config/link.ts

```diff
@@ -68,13 +68,20 @@ async function loadRemoteApp(localApp: AppInterface, apiKey: string | undefined)
   return app
 }
 
+interface NewAppConfig {
+  client_id?: string
+  name?: string
+  application_url?: string
+  webhook_api_version?: string
+}
+
 function mergeAppConfiguration(localApp: AppInterface, remoteApp: OrganizationApp): AppConfiguration {
-  const configuration = localApp.configuration || {}
+  const configuration: NewAppConfig = (localApp.configuration as unknown) || {}
 
   configuration.client_id = remoteApp.apiKey
   configuration.name = remoteApp.title
   configuration.application_url = remoteApp.applicationUrl
-  configuration.redirect_url_allowlist = remoteApp.redirectUrlWhitelist
+  configuration.webhook_api_version = '2023-04'
```

Comment:
This should be: `configuration.webhook_api_version = remoteApp.webhookApiVersion`?

Reply:
I think a bunch of fields are missing from the `OrganizationApp` data type, so it's currently missing